events/forms.py

```diff
@@ -0,0 +1,32 @@
+from django import forms
+from django.forms import ModelForm
+from .class_models.event_models import Event
+from .class_models.participant_model import EventParticipant
+from users.models import User
+
+
+class EventCreationForm(ModelForm):
+
+    def __init__(self, *args, **kwargs):
+        self.user_id = kwargs.pop('user_id')
+        super(EventCreationForm, self).__init__(*args, **kwargs)
+
+    title = forms.CharField(initial="", required=True, max_length=50)
+    location = forms.CharField(initial="", required=False, max_length=50)
+    description = forms.CharField(initial="", required=False)
+    date_time_start = date_time_end = forms.DateTimeField(
+        input_formats=['%I:%M %p %d-%b-%Y'],
```

Comment:
I can see you use the same input_format in more places
maybe you can define a constant with this value like: 
```suggestion
TIME_FORMAT=['%I:%M %p %d-%b-%Y']
...
        input_formats=TIME_FORMAT,
```

Reply:
done